docs/source/en/guides/cli.md

```diff
@@ -0,0 +1,445 @@
+
+Once you have your token, run the following command in your terminal:
+
+```bash
+>>> huggingface-cli login
+```
+
+This command will prompt you for a token. Copy-paste yours and press *Enter*. Then you'll be ask if the token should also be saved as a git credential. Press *Enter* again (default to yes) if you plan to use `git` locally. Finally, it will call the Hub to check that your token is valid and save it locally.
+
+```
+_|    _|  _|    _|    _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|_|_|_|    _|_|      _|_|_|  _|_|_|_|
+_|    _|  _|    _|  _|        _|          _|    _|_|    _|  _|            _|        _|    _|  _|        _|
+_|_|_|_|  _|    _|  _|  _|_|  _|  _|_|    _|    _|  _|  _|  _|  _|_|      _|_|_|    _|_|_|_|  _|        _|_|_|
+_|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
+_|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
+
+To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .
+Token: 
+Add token as git credential? (Y/n) 
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+Alternatively if you want to login without being prompt, you can pass the token from the command line. It is recommended to use an environment variable when doing so to avoid having your token pasted in your command history.
+
+```bash
+# Or using an environment variable
+>>> huggingface-cli login --token $HUGGINGFACE_TOKEN --add-to-git-credential 
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+## huggingface-cli whoami
+
+If you want to know if you are logged in, you can use `huggingface-cli whoami`. This command doesn't have any option and will simply print your username and the organizations you are part of on the Hub:
+
+```bash
+huggingface-cli whoami                                                                     
+Wauplin
+orgs:  huggingface,eu-test,OAuthTesters,hf-accelerate,HFSmolCluster
+```
+
+If you are not logged in, an error message will be printed.
+
+## huggingface-cli logout
+
+This commands logs you out. In practice, it will delete the token saved on your machine.
+
+<Tip warning={true}>
+
+This command will not log you out if you are logged in using the `HF_TOKEN` environment variable (see [reference](../package_reference/environment_variables#hftoken)). If that is the case, you must unset the environment variable in your machine configuration.
+
+</Tip>
```

Comment:
I don't think we need to make this a `<Tip>` since its important stuff we want users to read as part of the main content.